docs/reference/api/docker_remote_api_v1.23.md

```diff
@@ -2830,7 +2830,10 @@ Status Codes:
 
 Query Parameters:
 
-- **filters** - JSON encoded value of the filters (a `map[string][]string`) to process on the volumes list. There is one available filter: `dangling=true`
+- **filters** - JSON encoded value of the filters (a `map[string][]string`) to process on the volumes list. The filter value is one of:
+  -   `name=<volume-name>` Matches all or part of a volume name.
+  -   `dangling=true`  1/true or 0/false, return the volumes which are referenced or not by containers.
```

Comment:
Hm, thinking how we should describe this, perhaps;

```Markdown
When set to `true` (or `1`), returns all volumes that are "dangling"
(not in use by a container). When set to `false` (or `0`), only volumes
that are in use by one or more containers are returned.
```

Not really happy with that, so if anyone has better suggestions, please do!